package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "email-setup",
+  "version": "0.0.0",
+  "description": "Collection of utilities for checking email configuration settings.",
+  "main": "index.js",
+  "repository": "https://github.com/mixmaxhq/email-setup",
+  "author": "Trey Tacon <trey@mixmax.com> (https://mixmax.com)",
+  "license": "MIT",
+  "dependencies": {
+    "dmarc-parse": "^1.0.2",
+    "global": "^4.3.2",
+    "promise-callbacks": "^3.0.0",
+    "semver": "^5.4.1",
+    "spf-parse": "^1.0.4",
+    "underscore": "^1.8.3"
+  },
+  "devDependencies": {
```

Comment:
Can you enable `eslint` please? https://github.com/mixmaxhq/eslint-config-mixmax/tree/master/node